src/libstd/path.rs

```diff
@@ -1397,6 +1398,36 @@ impl From<String> for PathBuf {
     }
 }
 
+/// An error returned when parsing `Path` or `PathBuf` using `from_str` fails.
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[stable(feature = "rust1", since = "1.0.0")]
+pub struct ParsePathError { _priv: () }
+
+#[stable(feature = "rust1", since = "1.0.0")]
+impl fmt::Display for ParsePathError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        "provided string was not a valid path".fmt(f)
+    }
+}
+
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
The feature should not be `rust1`. Please create a feature name e.g. `path_from_str` and set the `since` to be `1.26.0`.